py/torch_tensorrt/dynamo/backend/lowering/_partition.py

```diff
@@ -14,6 +15,11 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_SINGLE_NODE_PARTITIONS: Set[str] = set(
+    "torch.ops." + str(module.new_operator)
```

Comment:
Do you know if theres a better type than string for this registry? Like is there a op type?

Reply:
I will look into this more - there is the `torch._ops.OpOverload` type which could be a substitute here.

Reply:
After looking more into this, `str` is likely the best choice for this registry for now. The issue is that node targets can be a mix of functions and `torch._ops.OpOverload` objects. For example, `torch.ops.aten.add.Tensor` is an overload object representing a Tensor addition op, whereas the operator for `get` is an actual Python function. The unifying framework which can connect all of these types is the [`_get_qualified_name`](https://github.com/pytorch/pytorch/blob/17166c2511fa8b18378438b4283f2d2b4625356e/torch/fx/node.py#L79) function, which can handle all of these types, and returns a string. I have updated the implementation here to use that function.